### PR TITLE
PR-6

### PR DESCRIPTION
**Pull request by fjpulidop**
Please, review my PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,4 @@
 - PR3
 - PR4
 - PR5
+- PR6
```
Comment (sergioromerocarrizosa) [inline README.md:7]: change to PR7

File: README.md [hunk 2]
```diff
@@ -4,3 +4,4 @@
 - PR3
 - PR4
 - PR5
```
Comment (jpulidop) [inline README.md:6]: I dont like this line. Remove!!

**Review by jpulidop — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fjpulidop) [reply README.md:7]: Ok..but no

**Review by sergioromerocarrizosa — APPROVED** [review]

**Review by jpulidop — APPROVED** [review]

**Comment by fjpulidop** [comment]
Integration changes for PR-6. Line removed.

**Pull request merged**